packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+
+import babelRequireHook from '@babel/register'
+import Listr from 'listr'
+import VerboseRenderer from 'listr-verbose-renderer'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+babelRequireHook({
+  extends: path.join(getPaths().api.base, '.babelrc.js'),
+  extensions: ['.js', '.ts', '.tsx', '.jsx'],
+  only: [getPaths().api.base],
+  plugins: [
+    [
+      'babel-plugin-module-resolver',
+      {
+        alias: {
+          src: getPaths().api.src,
+        },
+      },
+    ],
+  ],
+  ignore: ['node_modules'],
+  cache: false,
+})
+
+const { db } = require(path.join(getPaths().api.lib, 'db'))
+
+const runScript = async (scriptPath) => {
+  const script = await import(scriptPath)
+  const startedAt = new Date()
+  await script.default({ db })
+  const finishedAt = new Date()
+
+  return { startedAt, finishedAt }
+}
+
+export const command = 'run'
+export const description = 'Run your script'
+export const builder = (yargs) => {
+  yargs
+  .positional('name', {
+    description: 'The name of the script to run',
+    type: 'string',
+  }).epilogue(
+    `Also see the ${terminalLink(
+      'Redwood CLI Reference',
+      'https://redwoodjs.com/docs/cli-commands#up'
+    )}`
+  )
+}
+
+export const handler = async (args) => {
+  const scriptPath = path.join(getPaths().api.scripts, `${args.name}.js`)
+
+  if (!fs.existsSync(scriptPath)) {
+    console.info(c.error(`\nNo script file (${scriptPath}) exists.\n`))
+    process.exit(0)
+  }
+
+  const scriptTasks = [
+    {
+      title: 'Running script',
+      task: async () => {
+        try {
+          const { startedAt, finishedAt } = await runScript(scriptPath)
```

Comment:
Maybe console log the start time and finish time?

Reply:
So I actually get an output like so

```
$ /Users/viperfx/code/redwood-app/node_modules/.bin/rw run --name push
[09:09:22] Running script [started]
[09:09:24] Running script [completed]

Script has completed successfully.
```

Is this coming from the task wrapper?

Reply:
I'd suggest removing the `startedAt` and `finishedAt` variables returned from `runScript` in that case

Reply:
Ah ok - yep removed.